documentation/modules/overview/con-overview-components-cruise-control.adoc

```diff
@@ -0,0 +1,11 @@
+// This module is included in:
```

Comment:
Isn't this file the same as the con-metrics-overview-cruisecontrol file? Or does this horrible docs format make this mandatory?